docs/python/modelspec/morphology/MorphIO.rst

```diff
@@ -0,0 +1,93 @@
+.. _MorphIO:
+
+MorphIO
+=======
+
+MorphIO is a library for reading and writing neuron morphology files.
+It supports the following formats: SWC, ASC (aka. neurolucida) and H5 v1.
+
+We use MorphIO to provide a C++ implementation that will match Import3D_GUI HOC stack.
+This will allow us to read neuron morphology files in a variety of formats, with an increased performance.
+
+There will also be a backward compatible layer that will allow us to use the HOC stack together with MorphIO.
+Finally we will be able to update the Import3D_GUI HOC stack to use MorphIO so that CellBuilder/Import3d can use it. 
+
+Morphio is not available for Windows, because further support is required in MorphIO for the MinGW toolchain.
+
+.. _morphio_api:
+
+MorphIO Python API
+===============
+
+
+The `morphio_api` submodule provides a Python interface to the MorphIO morphology loading,
+which is used to load neuronal morphology data.
+
+Functions
+---------
+
+.. py:function:: morphio_load(obj, morph_file)
+
+    Load morphology data from a morphology file.
+
+    :param obj: A NEURON Cell object.
+    :param morph_file: A file path that contains morphology data.
+
+.. py:function:: morphio_load(obj, MorphIOWrapper)
+
+    Load morphology data from a morphology MorphIO wrapper.
+
+    :param obj: A NEURON Cell object.
+    :param MorphIOWrapper: A MorphIOWrapper object (see below).
+
+Classes
+-------
+
+.. py:class:: MorphIOWrapper
+
+    A Python binding for the MorphIOWrapper. This holds the MorphIO object and provides utility functions to access the data.
+
+    .. py:method:: __init__(self, path_or_file)
+
+        Initialize the `MorphIOWrapper` object with a file path or a Python object that contains morphology data.
+
+    .. py:method:: morph_as_hoc(self)
```

Comment:
Just `to_hoc` would be a bit more Pythonic IMHO

Reply:
Sounds great to me